compositor_render/src/render_loop.rs

```diff
@@ -82,6 +82,18 @@ pub(super) fn run_transforms(ctx: &RenderCtx, scene: &Scene) {
     }
 }
 
+pub(super) fn update_global_buffers(ctx: &RenderCtx, pts: Duration) {
```

Comment:
You would prefer to keep render_loop.rs without any logic for specific type of transformation. Can you move this method to shader.rs?

Also, it's not a strong opinion, but maybe name sth more generic? e.g. Shader::prepare_new_render_loop or sth like that? We could use consistent naming for transforms that also need to do sth at the beginning of the loop.

Reply:
So you're saying I should remove this function from `render_loop` and place it in `Shader`, but still call `Shader::prepare_new_render_loop` in the renderer? Do I understand correctly, or did you mean something else?

Reply:
yes